src/sss_client/pam_sss.c

```diff
@@ -995,7 +995,7 @@ static int eval_response(pam_handle_t *pamh, size_t buflen, uint8_t *buf,
                     break;
                 }
 
-                if (type == SSS_PAM_CERT_INFO && pi->cert_user == '\0') {
+                if (type == SSS_PAM_CERT_INFO && *pi->cert_user == '\0') {
```

Comment:
Both version are used in file  https://pagure.io/SSSD/sssd/blob/master/f/src/sss_client/pam_sss.c#_1499
We are not consistent here. Brackets make sense for operations where semantic might change and you do not want to remember which operator has higher priority (arithmetic operations, boolean operations ...)

But in this case semantic would not change if operator derefference `*` had higher priority then operator `->`. because you would not be able to compile code.
```
../sssd/src/sss_client/pam_sss.c: In function ‘prompt_sc_pin’:
../sssd/src/sss_client/pam_sss.c:1499:40: error: invalid type argument of ‘->’ (have ‘struct pam_items’)
     if (pi->token_name == NULL || (*pi)->token_name == '\0') {
                                        ^~
```